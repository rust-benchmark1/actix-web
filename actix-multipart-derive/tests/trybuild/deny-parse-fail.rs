use actix_multipart::form::MultipartForm;

#[derive(MultipartForm)]
#[multipart(duplicate_field = "no")]
struct Form {}

fn main() {}
