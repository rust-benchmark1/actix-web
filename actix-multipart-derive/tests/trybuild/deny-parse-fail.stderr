error: Unknown literal value `no`
 --> tests/trybuild/deny-parse-fail.rs:4:31
  |
4 | #[multipart(duplicate_field = "no")]
  |                               ^^^^
